README.md

```diff
@@ -1,2 +1,12 @@
 # hw01
 Repository for homework 01
+# Isaac LaGrand
+I am an MA student in International Relations.
```

Comment:
If you meant these to appear on separate lines, add a blank line. For instance

```
I am an MA student in International Relations.

My favorite non-fiction book is *Nature's Metropolis: Chicago and the Great West*

I am taking three courses this term. Ranked from Earliest to latest in the day they are:
```

Reply:
This becomes:

I am an MA student in International Relations.

My favorite non-fiction book is *Nature's Metropolis: Chicago and the Great West*

I am taking three courses this term. Ranked from Earliest to latest in the day they are: